src/lib/components/Modal/ModalContent.jsx

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { withGlobalProps } from '../../provider';
+import { isChildrenEmpty } from '../_helpers/isChildrenEmpty';
+import styles from './ModalContent.scss';
+
+export const ModalContent = ({
```

Comment:
If `ModalContent` is used without `ModalHead` and `ModalContent` (or in combination with `ModalHead` only), padding of the component is not centered:

<img width="653" alt="image" src="https://user-images.githubusercontent.com/10213870/175777488-56f8b010-1e1e-44d3-ade1-fd5e3140d9cc.png">

This is why I added `mt-4` into `Blocking modal without title`. @adamkudrna, I think we should add styles for all suitable combinations. Content, Head + Body (with Content), Content (with Content) + Footer. Are you able to add it into this PR? Or would you rather solve it in different PR?

Reply:
It's not much straightforward once `ModalBody` and `ModalContent` are separate components but this should work:

```scss
// ModalBody.scss

.root:first-child:not(.isRootScrollable) {
  padding-top: settings.$padding-y;
}
```

All other combinations are fine I think.